tpl/collections/init.go

```diff
@@ -122,6 +122,10 @@ func init() {
 					`<a href="https://www.google.com?{{ (querify "q" "test" "page" 3) | safeURL }}">Search</a>`,
 					`<a href="https://www.google.com?page=3&amp;q=test">Search</a>`,
 				},
+				{
+					`{{ (querify ["foo" 1 "bar" 2]) | safeHTML }}`,
```

Comment:
Square brackets are not allowed.  You have to use the `slice` fuction.  Try  `{{ slice "foo" 1 "bar" 2 | querify | safeHTML }}`.

The examples are tested in the `tpl/tplimpl` package.  I usually just do `go test ./tpl/...`

Reply:
@moorereason Thanks!

Yeah I know how to run the test, I'm asking is there a way to run querify to get the URL encoded query for the example, other than using it for a website and seeing the result?

Reply:
I meant how can I run a function to get the output? Say I want to see what will be the result if I run querify for a particular input, is there a way I can do that without adding it to a Hugo website and seeing what happens?

Reply:
It's still failing, the test passed locally, where am I going wrong?